src/routes/user-routes.ts

```diff
@@ -0,0 +1,28 @@
+import Elysia, { t } from "elysia";
+import { user } from "../controllers/userController";
+import { Set } from "../utils/utils";
```

Comment:
Please rename `Set` to something else, because it’s really confusing, JS has a `Set` collection, so treat that word as a reserved one